### PR TITLE
Add getActivityCostId method to Person

### LINKED ISSUE #116: Add method to get cost owed by a Person for a specific activityId to Person class
This method should return a double representing the cost owed by a Person for a specific Activity by specifying its activityId.

### PR DESCRIPTION
**Pull request by matheril**
This method accepts an int representing the activityId. It returns the cost owed by this Person for this Activity as a double.

This method throws InvalidDataException if the Person has no Activities, or if the activityId is not found in the Person's list of Activities.

Fixes #116

### CODE REVIEW

File: src/main/java/seedu/splitlah/data/Person.java [hunk 1]
```diff
@@ -67,6 +67,26 @@ public double getTotalCost() {
...
+     * Returns the cost owed by this Person object for an Activity.
+     *
+     * @param activityId int representing activityId of the Activity.
+     * @return double representing cost owed by the Person for this Activity.
+     * @throws InvalidDataException if this Person is not participating in any Activities
+     *         or the activityId is not found.
+     */
+    public double getActivityCostOwed(int activityId) throws InvalidDataException {
+        if (activityCostList.isEmpty()) {
+            throw new InvalidDataException("This person is not participating in any activities.");
```
Comment (warrencxw) [inline src/main/java/seedu/splitlah/data/Person.java:80]: Consider shifting all the message constants to Message class

**Review by warrencxw — CHANGES_REQUESTED** [review]
Minor changes can be made to improve the code.

File: src/main/java/seedu/splitlah/data/Person.java [hunk 2]
```diff
@@ -67,6 +67,26 @@ public double getTotalCost() {
...
+        }
+        for (ActivityCost i : activityCostList) {
```
[the remark this answers is not in the document]
Comment (matheril) [reply src/main/java/seedu/splitlah/data/Person.java]: Resolved

File: src/main/java/seedu/splitlah/data/Person.java (hunk 1 above)
Comment (matheril) [reply src/main/java/seedu/splitlah/data/Person.java]: Resolved

**Review by warrencxw — APPROVED** [review]
Looks good to merge.

**Pull request merged**